app/room.py

```diff
@@ -41,32 +42,27 @@ def __init__(self, Room_name = "",Room_type = ""):
 
 	"""
 	Allocate a person to a room at random depending on if the person 
-	wants allocation{ also yes for staff }. 
+	wants allocation { also yes for staff }. 
 	check if room is full.
-		NB
-			fellow are allocated either office or living space
-			staff are allocated only office.
 	"""	
-	def allocate_member_a_room(self,name = "",id_no = 0):
+	def allocate_member_a_room(self,name = "",id_no = 0, room_type = ""):
 		self.name = name
 		self.id_no = id_no
+		self.list = {"FELLOW":"LIVINGSPACE","STAFF":"OFFICE"}
+		self.room_type = room_type
 
 		#1. Add to the list of current occupents in the room instance
 
 		#value = "NAME :",self.name," ID :",self.id_no
-		self.Room_instance.current_members.append(self.name)
+		self.room_instance.current_members.append(self.name)
 
 		#2. Add person to the amity people info 
 		#KEY : ID number, VALUE : NAME
 
 		Storage.people_info[self.id_no] = self.name
 
-		# print self.name," added to the system.In room : ",self.Room_name
-		# for size in range(len(self.Room_instance.current_members)):
-		# 	print self.Room_instance.current_members[size]
-
-		print "Your room allocation has been successfully!!"
-		print "Your room space is in {}.".format(self.Room_name)
+		print "Your room allocation has been successfull!!"
+		print "Your {} space is in {}.".format(self.list[self.room_type],self.room_name)
 		print "Your ID number is {}.".format(self.id_no)
 		print "Keep it safe and Welcome to the Dojo!!\n"
```

Comment:
I saw it best to bring the system to something understandable for new comers. Thanks.